src/pages/_document.tsx

```
@@ -5,6 +5,18 @@ const Document = () => {
   return (
     <Html lang="en">
       <Head>
+         {/* Google Tag Manager Script */}
+         <script
+          dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)